back-end/__tests__/app.test.js

```diff
@@ -491,3 +491,71 @@ describe("/api/teams", () => {
       });
   });
 });
+
+describe.only("/api/responses", () => {
+  test("PUT 201: Should insert a response object into responses and return it", () => {
+    return request(app)
+      .put("/api/responses")
+      .send({
+        user_id: 5,
+        fixture_id: 3,
+        response: 2,
+      })
+      .expect(201)
+      .then(({ body }) => {
+        const { response } = body;
+        expect(response.response_id).toBe(9);
+        expect(response.user_id).toBe(5);
+        expect(response.fixture_id).toBe(3);
+        expect(response.response).toBe("2");
+        expect(typeof response.created_at).toBe("string");
+      });
+  });
+
+  test("PUT 201: Should update an existing response object from responses and return it", () => {
```

Comment:
These 2 tests are doing the same thing, perhaps remove one of them?
```
test("PUT 201: Should insert a response object into responses and return it"
test("PUT 201: Should update an existing response object from responses and return it
```

Reply:
They are doing slightly differnt things - the first one is for creating a new response in the db, while the second one is for updating an existing one.